testing/libfort/APKBUILD

```
@@ -0,0 +1,45 @@
+# Maintainer: Pete Dietl <petedietl@gmail.com>
+pkgname=libfort
+pkgver=0.1.3
+pkgrel=0
+pkgdesc="A simple crossplatform library to create formatted text tables."
+url="https://github.com/seleznevae/libfort"
+arch="all"
+license="MIT"
+makedepends="cmake binutils-gold"
+subpackages="$pkgname-dev"
+source="$pkgname-$pkgver.tar.gz::https://codeload.github.com/seleznevae/libfort/tar.gz/v$pkgver"
+builddir="$srcdir/$pkgname-$pkgver"
+
+prepare() {
+	default_prepare
+	mkdir -p "$builddir/build"
+}
+
+build() {
+	cd "$builddir/build"
```

This is currently the de facto way of making and cd'ing into build dir.
```suggestion
	mkdir build && cd build
```

---

hmm should I assume that `$builddir` is already created and that I am in that directory?
I was trying to have a build directory in a build directory to do an out-of-source build. I'm not sure what the best way to do that is.

---

builddir is the directory where source is extracted. So build is at the same level as the src files.
It is the common way to doing and "out-of-source build".